### DISCUSSION TITLE
React app on vercel.app not working

### DISCUSSION THREAD

**Discussion by shiloh92** [vercel/community#1854] [Help] [answered]
### Summary

Based on guidance available for React apps, I am finding difficulties using components due to duplicate imports?
I have no other imports and keep getting these errors on the default React template available on Vercel.app
Even a simple Hello world button errors out modifying the standard template.

I've attempted to get basic vercel app sites up before and Vercel.app seems to be extremely difficult to use -- as there is no indication where the error comes from.

**Comment by amyegan** [comment, marked as answer]
Hi @shiloh92. You should be able to find more error details in [Build Logs](https://vercel.com/docs/concepts/deployments/logs) and [Runtime Logs](https://vercel.com/docs/concepts/observability/runtime-logs#). If you can share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) and a link to the [template](https://vercel.com/templates?search=react) you used, I would be happy to take a look and offer some advice
